chapters/appendix-V-using-SPDX-short-identifiers-in-source-files.md

```diff
@@ -24,9 +24,9 @@ The SPDX License Identifier syntax may consist of a single license (represented
 
 The tag should appear on its own line in the source file, generally as part of a comment.
 
-SPDX-License-Identifier: <SPDX License Expression>
+SPDX-License-Identifier: \<SPDX License Expression\>
 
-## Representing Single License
+### Representing Single License
 
 A single license is represented by using the short identifier from [SPDX license list](https://spdx.org/licenses/), optionally with a unary "+" operator following it to indicate "or later" versions may be applicable.
```

Comment:
I accidentally read this. Aren't license identifiers with `+` deprecated, at least the well-known GPLs?

Reply:
As far as I understand, `+` is deprecated *only* for GPL, and not for any other licenses. `CC-BY-SA-3.0+` is valid SPDX. Correct me if I'm wrong.

Reply:
"+" is valid syntax on other licenses.    I spotted an old style GPL reference that was in the prior version, and went in and updated it.